BitwardenShared/Core/Auth/Repositories/AuthRepository.swift

```diff
@@ -63,6 +63,11 @@ protocol AuthRepository: AnyObject {
     ///
     func getFingerprintPhrase() async throws -> String
 
+    /// Gets the organization identifier by email for the single sign on process.
+    /// - Parameter email: The email to get the organization identifier.
+    /// - Returns: The organization identifier, if any that complies with verified domains. Also `nil` if empty.
+    func getSingleSignOnOrganizationIdentifierBy(email: String) async throws -> String?
```

Comment:
⛏️ Minor, but I feel like you could leave off `By` from the method name. If anything, conventions in Swift would use `by` as the argument label (e.g. `getSingleSignOnOrganizationIdentifier(by email: String)`, but I think `getSingleSignOnOrganizationIdentifier(email:)` reads the best.